package.json

```diff
@@ -12,6 +12,8 @@
     "test:debug": "ng test --browsers ChromeDebug",
     "lint": "ng lint",
     "e2e": "protractor ./e2e/protractor.conf.js",
+    "screenshot-testing": "protractor ./e2e/protractor.conf.js --specs=e2e/features/screenshot-testing.feature",
```

Comment:
please rename this to `test:screenshot` and `test:screenshot-update-goldens`

Reply:
I renamed scripts to suggested ones